hazelcast/src/main/java/com/hazelcast/kubernetes/KubernetesApiProvider.java

```diff
@@ -51,14 +51,14 @@ default Integer extractPort(JsonValue subsetJson) {
         JsonArray ports = toJsonArray(subsetJson.asObject().get("ports"));
         for (JsonValue port : ports) {
             JsonValue hazelcastServicePort = port.asObject().get("name");
-            if (hazelcastServicePort != null && hazelcastServicePort.asString().equals("hazelcast-service-port")) {
+            if (hazelcastServicePort != null && hazelcastServicePort.asString().equals("hazelcast")) {
                 JsonValue servicePort = port.asObject().get("port");
                 if (servicePort != null && servicePort.isNumber()) {
                     return servicePort.asInt();
                 }
             }
         }
-        if (ports.size() > 0) {
+        if (ports.size() == 1) {
```

Comment:
Why changed in this way `equals to 1`

Reply:
Anyway, It is not a big deal. In the best practice, we assume to have a port named as `hazelcast`.